applications/appunorganized/appunorganized-plugins/org.csstudio.swt.rtplot/src/org/csstudio/swt/rtplot/internal/PlotProcessor.java

```diff
@@ -84,12 +84,32 @@ public ValueRange call() throws Exception
                         {
                             final PlotDataItem<XTYPE> item = data.get(i);
                             final double value = item.getValue();
-                            if (! Double.isFinite(value))
-                                continue;
-                            if (value < low)
-                                low = value;
-                            if (value > high)
-                                high = value;
+                            final double max = item.getMax();
+                            final double min = item.getMin();
+                            
```

Comment:
We could do with a comment explaining what's going on here.

Reply:
I have just added comments to the code to describe how the min, max or value are used to determine the range. Please see latest [commit](https://github.com/ControlSystemStudio/cs-studio/pull/2663/commits/67d15b079ae937c455c507015efcb9cd7a52d901).